content/600-about/200-prisma-docs/20-style-guide/05-prisma-product-names.mdx

```diff
@@ -5,17 +5,48 @@ metaDescription: 'This section of the style guide provides guidelines on how to
 tocDepth: 3
 ---
 
-## Use the following forms
+Use the official Prisma product names and feature names listed below in documentation. Do not abbreviate any of these names, except where noted.
+
+## Product names
 
 - The Prisma Data Platform (or just "PDP" - no definite article - after the first mention)
 - Prisma Client (not any variation of "the Client")
 - Prisma Schema
 - Prisma Migrate
 - Prisma CLI
 - Prisma Studio
+
+## Feature names
+
 - Data Proxy (_not_ Prisma Data Proxy)
 - Data Browser (in the Prisma Data Platform UI)
 - Query Console (in the Prisma Data Platform UI)
 - Schema Viewer (in the Prisma Data Platform UI)
 
-Do not abbreviate any of these names, except where noted above.
+## Use of articles with product names and feature names
+
+Do not use a definite article in front of official product names.
```

Comment:
This is not strictly true - IMO we should use the list of specific product names above. We do use a definite article for the Prisma Data Platform (but not when we abbreviate it to PDP). In future, Prisma might introduce more products, which might use definite articles, so I think it's easier just to refer to the list above.

Reply:
I've left the instance of "The Prisma Data Platform" above but I am not sure I wholeheartedly agree with it.

If it is a product name, I think the rule of not using "the" should apply to it. If we were to say something like "the Data Platform", then that kind of makes sense, but with the note that it is unambiguously clear from the context that we're talking about PDP and not another data platform and this is only used for brevity purposes.

From previous experience, long-form product names (Prisma Data Platform) are introduced on first mention in a document, and then their short-form version is used consistently (PDP). This means that any additional forms (the Data Platform, Data Platform, data platform, the platform) are avoided and discouraged to avoid any confusion.